.github/workflows/shinyapps_deploy.yml

```diff
@@ -8,7 +8,7 @@ on:
     branches:
       - IT-828
       - master
-      - release*
+      - staging
```

Comment:
```suggestion
      - release*
      - staging
```

Reply:
We would need to keep both to allow releases to be deployed to live app

Reply:
I havent worked on the deployment workflow in a bit, so I need to revisit this to make sure all contingencies of deployment are maintained. 

@andrewelamb would you be okay to revert this commit and hold this for a separate PR?